pandas/core/indexing.py

```diff
@@ -2058,6 +2058,8 @@ def ravel(i):
             # we have a frame, with multiple indexers on both axes; and a
             # series, so need to broadcast (see GH5206)
             if sum_aligners == self.ndim and all(is_sequence(_) for _ in indexer):
+                if is_empty_indexer(indexer[0], ser._values):
+                    return ser._values.copy()
```

Comment:
evidently it works, but this seems like a weird place to handle this.  is my intuition wrong here?

Reply:
This ensures, that an indexer like ``(np.array([]), np.array([1]))`` is handled the same as ``(np.array([]), 1)``, hence I handled it here.